openstack/cinder/values.yaml

```diff
@@ -57,6 +57,7 @@ backend_defaults:
   max_over_subscription_ratio: 1.5
 
 backends:
+    enabled: vmware
```

Comment:
Just out of curiosity: Am I reading this correct, that the default before was `vmware,standard_hdd` and is now only `vmwware` unless overridden by a value from secrets?

Reply:
I'm simply overly-cautious and thinking about what happens if the change in secrets is (for some reason) not applied at the same time.

Reply:
Yes, you are correct.   standard_hdd is a new backend that hasn't been deployed to production.   The requirements of deploying this backend has changed to only wanting to deploy it to very specific regions until we have more customers asking for it.  That's why it was removed and added as an optional configurable change here as well as secrets.

Reply:
I see. Thanks!